packages/shared/src/components/Sidebar.tsx

```diff
@@ -106,16 +114,36 @@ const NavItem = classed(
 
 export default function Sidebar(): ReactElement {
   const { openSidebar, toggleOpenSidebar } = useContext(SettingsContext);
+  const { feedSettings } = useFeedSettings();
+  const { alerts, updateAlerts } = useContext(AlertContext);
   const [showSettings, setShowSettings] = useState(false);
+  const [isFilterOpen, setIsFilterOpen] = useState(false);
+
+  const showFilters = () => {
+    if (
+      alerts?.filter &&
+      (feedSettings?.includeTags?.length ||
+        feedSettings?.blockedTags?.length ||
+        feedSettings?.excludeSources?.length ||
+        feedSettings?.advancedSettings?.length)
+    ) {
+      updateAlerts({ filter: false });
+    }
```

Comment:
We are now loading the complete feedSettings here, maybe we should move this to the opening of the feedFilter as that will already load them as well?
What do you think?

Note: We can keep the alerts context for the dot.

Reply:
I agree. This is the better way. I will work on this in the upcoming commits.